src/smt/smt_engine.cpp

```diff
@@ -1653,7 +1653,7 @@ void SmtEngine::declareSynthFun(const std::string& id,
     setUserAttribute("sygus-synth-grammar", func, attr_value, "");
   }
   Trace("smt") << "SmtEngine::declareSynthFun: " << func << "\n";
-  Dump("raw-benchmark") << SynthFunCommand(id, func, sygusType, isInv, vars);
+  // Dump("raw-benchmark") << SynthFunCommand(id, func, sygusType, isInv, vars);
```

Comment:
Can you add a comment that this is broken and which links this github projects issue https://github.com/CVC4/cvc4-projects/issues/211 ?  (I just added it for this purpose).

Reply:
Done!